### PR TITLE
Feat/release 2 q1 2020

### PR DESCRIPTION
**Pull request by repjarms**
### Summary

This adds new features to the Swift SDK, based on updated API definitions:

## Watson AssistantV1/V2
- Adds `includeAudit` parameter to most operations. This had traditionally been supported by the service but was only in the API definitions for a limited set of operations.
- Updates the `RuntimeEntity` models for added detail.

## VisualRecognitionV4
- Add Object Metadata operations. These operations allow users to manipulate objects in their trained models. CRUD operations have been added.

### CODE REVIEW

File: Source/AssistantV2/Models/MessageContextSkills.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2020.
+ * (C) Copyright IBM Corp. 2018, 2019.
```
Comment (mkistler) [inline Source/AssistantV2/Models/MessageContextSkills.swift]: Hmmm ... should not change copyright here.

File: Source/AssistantV2/Models/SearchResultHighlight.swift [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2019, 2020.
+ * (C) Copyright IBM Corp. 2019.
```
Comment (mkistler) [inline Source/AssistantV2/Models/SearchResultHighlight.swift]: This looks odd -- should be investigated.  Was the 2020 date in the original copyright wrong?

File: Source/CompareComplyV1/CompareComply.swift [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2020.
+ * (C) Copyright IBM Corp. 2018, 2019.
```
Comment (mkistler) [inline Source/CompareComplyV1/CompareComply.swift]: Was 2020 wrong?

File: Source/DiscoveryV1/Models/AggregationResult.swift [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2020.
+ * (C) Copyright IBM Corp. 2018, 2019.
```
Comment (mkistler) [inline Source/DiscoveryV1/Models/AggregationResult.swift]: Was 2020 wrong?

File: Source/NaturalLanguageUnderstandingV1/Models/MetadataOptions.swift [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2017, 2020.
+ * (C) Copyright IBM Corp. 2017, 2019.
```
Comment (mkistler) [inline Source/NaturalLanguageUnderstandingV1/Models/MetadataOptions.swift]: Was 2020 wrong?

File: Source/SpeechToTextV1/Models/SpeechRecognitionAlternative.swift [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2016, 2020.
+ * (C) Copyright IBM Corp. 2016, 2019.
```
Comment (mkistler) [inline Source/SpeechToTextV1/Models/SpeechRecognitionAlternative.swift]: Was 2020 wrong?

File: Source/SpeechToTextV1/SpeechToText+Recognize.swift [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2018, 2020.
```
Comment (mkistler) [inline Source/SpeechToTextV1/SpeechToText+Recognize.swift]: Was 2020 wrong?

File: Source/SpeechToTextV1/WebSockets/SpeechToTextSession.swift [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2016, 2019.
+ * (C) Copyright IBM Corp. 2016, 2020.
```
Comment (mkistler) [inline Source/SpeechToTextV1/WebSockets/SpeechToTextSession.swift]: Was 2020 wrong?

I'll stop now with this comment, but you should check all these.

**Comment by repjarms** [comment]
@mkistler I believe that the 2020 dates were incorrect from the previous release. At that time, you mentioned that they should be resolved correctly in the next PR. This is an attempt to fix those.

**Pull request merged**